vcl_templates/www.vcl.erb

```diff
@@ -235,6 +235,21 @@ sub vcl_recv {
     }
   }
 
+  # Rules for GOVUK-ABTest-Benchmarking
+  if (req.url ~ "[\?\&]ABTest-Benchmarking=A(&|$)") {
+    set req.http.GOVUK-ABTest-Benchmarking = "A";
+  } else if (req.url ~ "[\?\&]ABTest-Benchmarking=B(&|$)") {
+    set req.http.GOVUK-ABTest-Benchmarking = "B";
+  } else if (req.http.Cookie ~ "ABTest-Benchmarking") {
+    # Set the value of the header to whatever decision was previously made
+    set req.http.GOVUK-ABTest-Benchmarking = req.http.Cookie:ABTest-Benchmarking;
+  } else {
+    # We default everyone to A, because remote testers will have a param in the
+    # URL to add them to the B group. In this test, we don't want anyone else
+    # having the B variant.
+    set req.http.GOVUK-ABTest-Benchmarking = "A";
```

Comment:
I'm tempted to remove this last `else` and not assign any cookie/header to non-testers. This way, we don't `pollute` the cookies of most GOV.UK users. Thoughts?